selenium_tests/ab_tool/regression_suite.py

```diff
@@ -0,0 +1,37 @@
+"""
+To run these tests from the command line in a local VM, you'll need to set up the environment:
+> export PYTHONPATH=/home/vagrant/icommons_lti_tools
+> export DJANGO_SETTINGS_MODULE=icommons_lti_tools.settings.local
+> sudo apt-get install xvfb
+> python selenium_tests/regression_tests.py
+
+Or for just one set of tests, for example:
+> python selenium_tests/manage_people/mp_test_search.py
+
+In PyCharm, if xvfb is installed already, you can run them through the Python unit test run config
+(make sure the above environment settings are included)
+"""
+
+import unittest
+import time
+from selenium_tests.ab_tool.ab_tests import test_ab_tool
+import HTMLTestRunner
+
+
+dateTimeStamp = time.strftime('%Y%m%d_%H_%M_%S')
```

Comment:
style nitpick: python usually prefers underscore_case to camelCase for anything not a class name.